src/main/java/io/openepi/crop_health/ApiClient.java

```diff
@@ -905,7 +905,7 @@ public <T> T deserialize(Response response, Type returnType) throws ApiException
             // ensuring a default content type
             contentType = "application/json";
         }
-        if (isJsonMime(contentType)) {
+        if (isJsonMime(contentType) || contentType.startsWith("text/plain")) {
```

Comment:
Why is this needed? Are we returning the wrong content-type from the CropHealt API?

Reply:
Did not think to question that 😅 But yes, I think it returns the wrong type:
<img width="644" alt="image" src="https://github.com/user-attachments/assets/93d5372a-f51c-431b-8236-81d2e1e2b284">

Reply:
Yes, we should probably fix it in the crop-health API instead of doing this here.